src/main/java/com/minecolonies/coremod/commands/ActionMenuType.java

```diff
@@ -0,0 +1,305 @@
+import com.minecolonies.coremod.commands.citizencommands.CitizenInfoCommand;
+import com.minecolonies.coremod.commands.citizencommands.KillCitizenCommand;
+import com.minecolonies.coremod.commands.citizencommands.ListCitizensCommand;
+import com.minecolonies.coremod.commands.citizencommands.RespawnCitizenCommand;
+import com.minecolonies.coremod.commands.colonycommands.AddOfficerCommand;
+import com.minecolonies.coremod.commands.colonycommands.ChangeColonyOwnerCommand;
+import com.minecolonies.coremod.commands.colonycommands.ColonyTeleportCommand;
+import com.minecolonies.coremod.commands.colonycommands.DeleteColonyCommand;
+import com.minecolonies.coremod.commands.colonycommands.DisableBarbarianSpawnsCommand;
+import com.minecolonies.coremod.commands.colonycommands.DoRaidNowCommand;
+import com.minecolonies.coremod.commands.colonycommands.DoRaidTonightCommand;
+import com.minecolonies.coremod.commands.colonycommands.HomeTeleportCommand;
+import com.minecolonies.coremod.commands.colonycommands.ListColoniesCommand;
+import com.minecolonies.coremod.commands.colonycommands.MakeNotAutoDeletableCommand;
+import com.minecolonies.coremod.commands.colonycommands.RefreshColonyCommand;
+import com.minecolonies.coremod.commands.colonycommands.ShowColonyInfoCommand;
+import com.minecolonies.coremod.commands.colonycommands.requestsystem.RSResetAllCommand;
+import com.minecolonies.coremod.commands.colonycommands.requestsystem.RSResetCommand;
+import com.minecolonies.coremod.commands.generalcommands.BackupCommand;
+import com.minecolonies.coremod.commands.generalcommands.CheckForAutoDeletesCommand;
+import com.minecolonies.coremod.commands.generalcommands.RaidAllNowCommand;
+import com.minecolonies.coremod.commands.generalcommands.RaidAllTonightCommand;
+import com.minecolonies.coremod.commands.generalcommands.RandomTeleportCommand;
+import com.minecolonies.coremod.commands.generalcommands.ScanCommand;
+import com.minecolonies.coremod.commands.generalcommands.WhereAmICommand;
+import com.minecolonies.coremod.commands.generalcommands.WhoAmICommand;
+import com.minecolonies.coremod.commands.killcommands.AnimalKillCommand;
+import com.minecolonies.coremod.commands.killcommands.BarbarianKillCommand;
+import com.minecolonies.coremod.commands.killcommands.ChickenKillCommand;
+import com.minecolonies.coremod.commands.killcommands.CowKillCommand;
+import com.minecolonies.coremod.commands.killcommands.MobKillCommand;
+import com.minecolonies.coremod.commands.killcommands.PigKillCommand;
+import com.minecolonies.coremod.commands.killcommands.SheepKillCommand;
+
+//PMD.AvoidDuplicateLiterals: We want to have literals used instead of constants as we are defining commands
+//and do not necessarily want one command's syntax dependent on another command
+//PMD.ExcessiveImports: This class DOES have a high degree of coupling by design.
+@SuppressWarnings({"PMD.AvoidDuplicateLiterals", "PMD.ExcessiveImports"})
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Suppressing the 'PMD.AvoidDuplicateLiterals, PMD.ExcessiveImports' warnings is not allowed [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1309)